### PR TITLE
Kovalenko: lesson04

### PR DESCRIPTION
**Pull request by alexkoval6030**
close: #21

### CODE REVIEW

File: kovalenko/lesson04/web-CV/pom.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                    <version>2.5.2</version>
+                </plugin>
+                <plugin>
+                    <artifactId>maven-deploy-plugin</artifactId>
+                    <version>2.8.2</version>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
+</project>
```
Comment (rakovets) [inline kovalenko/lesson04/web-CV/pom.xml:77]: "Violation: POSIX standard defines a line: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206
Solution: Add terminating <newline>"

File: kovalenko/lesson04/web-CV/src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns="https://jakarta.ee/xml/ns/jakartaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-app_5_0.xsd"
+         version="5.0">
+</web-app>
```
Comment (rakovets) [inline kovalenko/lesson04/web-CV/src/main/webapp/WEB-INF/web.xml:6]: "Violation: POSIX standard defines a line: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206
Solution: Add terminating <newline>"

File: kovalenko/lesson04/web-CV/src/main/webapp/resources/css/style.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+.about-me-photo {
+    width: 40%;
+    float: left;
+}
+
+.about-me-contact {
+    width: 60%;
+    margin-top: 3%;
+    float: left;
+}
```
Comment (rakovets) [inline kovalenko/lesson04/web-CV/src/main/webapp/resources/css/style.css:72]: "Violation: POSIX standard defines a line: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206
Solution: Add terminating <newline>"

**Review by rakovets — CHANGES_REQUESTED** [review]
You need to remove build artifacts (directory^ `target`).

File: kovalenko/lesson04/web-CV/pom.xml (hunk 1 above)
Comment (alexkoval6030) [reply kovalenko/lesson04/web-CV/pom.xml:77]: correct

File: kovalenko/lesson04/web-CV/src/main/webapp/WEB-INF/web.xml (hunk 2 above)
Comment (alexkoval6030) [reply kovalenko/lesson04/web-CV/src/main/webapp/WEB-INF/web.xml:6]: correct

File: kovalenko/lesson04/web-CV/src/main/webapp/resources/css/style.css (hunk 3 above)
Comment (alexkoval6030) [reply kovalenko/lesson04/web-CV/src/main/webapp/resources/css/style.css:72]: correct

**Review by rakovets — APPROVED** [review]
Approved

**Pull request merged**